comm/src/main/protobuf/coop/rchain/comm/protocol/p2p.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+package coop.rchain.comm.protocol.rchain;
+
+import "scalapb/scalapb.proto";
+
+option (scalapb.options) = {
+  package_name: "coop.rchain.comm.protocol.rchain"
+  flat_package: true
+};
+
+message Handshake {
+    uint64 number = 1;
+}
+
+message HandshakeResponse {
```

Comment:
Is Handshake.number a unique id?  If so, don't we want something in HandshakeResponse to tell which Handshake it's responding to?

Reply:
Ah, sorry. That is there to ensure that this is actually working. Without it, you get an empty serialization, which wasn't telling me much. I'll add some commentary.